docs/internal.rst

```diff
@@ -323,6 +328,9 @@ Following code is taken from ``clojure.c`` (with modifications).
 ``parent``, values stored to ``scope [0]`` and ``scope [1]`` are all
 kind of strings.
 
+.. TODO: ... all kind of strings.
+	Q to @masatake-san: I cannot understand this sentence.
+
```

Comment:
This should  be:

```
``paernt``, values stored to ``scope [0]`` and ``scope[1]`` are all C strings.
```

Reply:
Do I make sense?

Reply:
Now I understand.

But I think it's a little grammatically wrong. "parent"` and "value" are the subjects.
It should be 

> ``parent``, ``scope [0]`` and ``scope[1]`` are all C strings.

And I think it would be better to replace "C strings"  by "vStrings".

BTW do we really need this sentence?

Reply:
You are correct. I think it is better to keep the sentence for presenting the advantage of cork API.

I would like to extend the sentence you fixed like:
---

``parent``, ``scope [0]`` and ``scope[1]`` are all vStrings. The parser code must manage the life cycles of vStrings; the parser cannot free the vStrings till the tag referring the vStrings via its scope fields is printed and must free them after printing.

Reply:
Now I fully understand your intentions.

Reply:
Thank you refining.